### DISCUSSION TITLE
CORS issues on files in Next.js

### DISCUSSION THREAD

**Discussion by LouVerdun** [swellstores/community#136] [Help]
Hello everyone,

I'm having issues with CORS policies trying to access a file from Swell (.glb) file in Next.js.
I'm getting a `No 'Access-Control-Allow-Origin' header is present on the requested resource`.
Do you have an idea on how to overcome this situation as we don't have access to the CDNs configuration?

Thank you very much!

**Comment by maarten2424** [comment]
Hey @LouVerdun !
How are you trying to load that `.glb` file in Next.js? I'm assuming you are loading a 3D model?

**Comment by cmcollin41** [comment]
@LouVerdun did you figure this out? I started having cors issues when I started using nextjs 13 with app dir.

**Comment by mustafaswell** [comment]
Looks like @cmcollin41 solution here might be of help? https://github.com/orgs/swellstores/discussions/150

**Comment by decorsstudio** [comment]
I am having issue uploading the `.glb` file from swell dashboard as product media, It is not uploading or showing any error.
I want to know, Is swell accepting 3D file, it has this feature or not? as for my project it is top priority feature so I need to clarify it first to continue with Swell!!
cc: [logeshswell](https://github.com/logeshswell)

**Reply by logeshswell** [reply]
@decorsstudio We don't have any restrictions in uploading .glb files as product media.
But, referring to this [comment](https://github.com/orgs/swellstores/discussions/263#discussioncomment-6090770),

> There is a size limit of 10MB per file currently.
